surveys/2021-annual-survey/questions.md

```diff
@@ -706,6 +720,7 @@ Select one:
 - I'm not worried
 
 > Alternative: could rephrase as a 'select all that apply' question
+> Other Alternative: Could be structured as a ranking question: Rank the following...
```

Comment:
I'd like for this document to move this document more towards a "done" state and adding alternatives doesn't seem to do that. Perhaps moving these alternative possibilities to issues would be best.